celery/security/certificate.py

```diff
@@ -2,6 +2,7 @@
 import datetime
```

Comment:
```suggestion
from __future__ import annotations

import datetime
```

Reply:
That's not quite what is written on that answer... That's only about postponed evaluation of annotations.

So instead of:
```python
    def verify(self, data: bytes, signature: bytes, digest: Union["HashAlgorithm", "Prehashed"]) -> None:
```
I would write:
```python
    def verify(self, data: bytes, signature: bytes, digest: Union[HashAlgorithm, Prehashed]) -> None:
```

Reply:
What I meant of course is that this import would not be necessary if we only supported 3.8+.

Reply:
I see the benefits now: https://lukasz.langa.pl/61df599c-d9d8-4938-868b-36b67fdb4448/